src/pages/Mypage/Mypage.js

```diff
@@ -1,9 +1,167 @@
+  }
+  openMyBlog = () => {
+    this.setState({
+      openMyBlog: true,
+    });
+  };
+  openLikeBlog = () => {
+    this.setState({
+      openMyBlog: false,
+    });
+  };
+
+  handleUserData = () => {
+    fetch('./data/myinfo.json')
+      .then(res => res.json())
+      .then(data =>
+        this.setState({
+          userInfo: data,
+        })
+      );
+  };
+
+  componentDidMount() {
+    this.handleUserData();
+
+    fetch('./data/blogmock.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          myBlogList: data,
+        });
+      });
+    fetch('./data/likeblogmock.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          likeBlogList: data,
+        });
+      });
+    fetch('/data/recentblogmock.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          recentBlogList: data,
+        });
+      });
+  }
+
   render() {
-    return <div />;
+    const { openMyBlog, myBlogList, likeBlogList, recentBlogList, userInfo } =
+      this.state;
+    const { userName, userIntro, userImg } = userInfo;
+
+    return (
+      <div className="myPage">
+        <main>
+          <div className="myPageCover" />
+          <div className="myPageWrap">
+            <div className="myImg">
+              <img className="thumb" src={userImg} alt="thumb" />
+            </div>
+            <div className="myProfile">
+              <div className="myUser">{userName}</div>
+              <div className="myUserIntro">{userIntro}</div>
+              <dl className="follower">
+                <dd>
+                  <em>구독자</em>
+                  <span>0</span>
+                </dd>
+                <dd>
+                  <em>관심작가</em>
+                  <span>1</span>
+                </dd>
+                <div className="btnMore">
+                  <Button text="글쓰기" style={true} />
+                  <img className="more" src="./images/mymore.png" alt="more" />
+                </div>
+              </dl>
+            </div>
+          </div>
+        </main>
+        <div className="blogLike">
+          <button
+            onClick={this.openMyBlog}
+            className={openMyBlog ? 'activeMyBlogBtn' : 'inActiveMyBlogBtn'}
+          >
+            내가 쓴 글
+          </button>
+          <button
+            onClick={this.openLikeBlog}
+            className={openMyBlog ? 'inActiveLikeBlogBtn' : 'activeLikeBlogBtn'}
+          >
+            Recent & Like
+          </button>
+        </div>
+        {openMyBlog ? (
+          <div className="myBlog">
+            <ul>
+              {myBlogList.map(blog => {
+                return (
+                  <Bloglist
+                    key={blog.id}
+                    title={blog.title}
+                    subtitle={blog.subTitle}
+                    articlecontent={blog.articleContent}
+                    img={blog.img}
+                  />
+                );
```

Comment:
```suggestion
                return (
                  <Bloglist
                    key={blog.id}
                    blogdata={blog}
                  />
                );
```
위와 같이 데이터 전체를 넘겨준 후 자식 컴포넌트에서 구조분해할당을 할 시 this.props.blogdata와 같은 형태로 진행하시면 됩니다!